### DISCUSSION TITLE
My account is disabled.

### DISCUSSION THREAD

**Discussion by mukulsharma2212** [vercel/community#3744] [Account Issue]
### Question

My account is disabled when I logged in it. It show error you are account is disabled! Please Reply me 🙏

**Comment by amyegan** [comment]
Due to the sensitive nature of account infomation, we cannot safely discuss it in this public forum. Please email [registration@vercel.com](mailto:registration@vercel.com) and include the details of the project you want to deploy.

**Reply by mukulsharma2212** [reply]
Ok
